src/domains/Condition/components/ConditionTemplates/FileUploadTemplate/FileUploadTemplate.js

```diff
@@ -15,8 +15,8 @@ function FileUploadTemplate(props) {
 
   return (
     <Row noGutters mb={2}>
-      <Col cw={6}>
-        <CustomUploaderBox pl={2} mr={4}>
+      <Col cw={6} style={{ paddingRight: '32px' }}>
```

Comment:
Col has paddingRight property

Reply:
but when u use pr with noGutters in row, it doesn`t work